add key considerations (#2)
lsst/rtn-091: body.tex

\item The application layer covered in \secref{sec:application}
\end{enumerate}

\section{Key Considerations}
Before discussing the network infrastructure used for the Rubin Observatory, it is important to understand the conditions under which the Vera C. Rubin Observatory—and most observatories worldwide—operate. 

> Observatories typically have one or two operational sites: one in a city, where offices are located, and another where the actual observatory is situated. This latter site is by far the most complex to design and operate.

Reviewer: ```suggestion
Observatories typically have two operational sites: one in a city, where offices are located, and another where the actual observatory is situated. This latter site is by far more complex to design and operate.
```

> At Rubin Observatory, we have multiple operational sites. There are two sites in La Serena, one in Tucson, and a data center at Stanford. However, the most challenging site to design, and the one requiring the highest level of precaution, is located on the mountain, one of the two sites of La Serena. This site is not only subject to unique climatic conditions—situated over 2,600 meters above sea level with lower oxygen levels and little humidity—but is also difficult to access.

Reviewer: ```suggestion
At Rubin Observatory, we have multiple operational sites. There are two sites in La Serena, one in Tucson, and a data center at Stanford. However, the most challenging site to design, and the one requiring the highest level of precaution, is located on the mountain, one of the two sites in La Serena. This site is not only subject to unique climatic conditions—situated over 2,600 meters above sea level with lower oxygen levels and little humidity—but it is also difficult to access.
```

Therefore, when designing the computing infrastructure of the observatory, the following key conditions had to be considered:

\begin {itemize}
\item Harsh environmental conditions. As a direct consequence, hardware failures are inevitable, making it essential for the observatory to be well-prepared for such incidents.
\item Unreliable power grid. Power outages are almost unavoidable without a dedicated power station near the observatory. 
\item Limited accessibility in urgent situations. The time required for an engineer to reach the site after an alert is triggered must be considered. At Rubin Observatory, this response time is at least three hours, which can be critical for hardware failures. While some observatories mitigate this challenge by implementing shift-based staffing, vendor support remains constrained by the same logistical barriers. As a result, shipping and delivery of replacement parts can take several days.

> \item Human factor. The difficulty in accessing the remote site, low oxygen levels, and the pressure to resolve issues quickly can affect the performance of the staff and increase the risk of errors. Therefore, solutions should be as simple as possible, requiring minimal effort and decision-making from the staff.

Reviewer: ```suggestion
\item Human factor. The difficulty in accessing the remote site, low oxygen levels, and the pressure to resolve issues quickly can affect the performance of the staff and increase the risk of errors. Therefore, solutions should be as simple as possible, requiring minimal effort and on the spot decision-making from the staff.
```

Reviewer: I think we involve staff in decison making very well - but some decisions are agreed up front

Author: Yes, the idea is to consider automatic "fixes," such as redeploying from scratch with minimum intervention.
